src/test/rustdoc/inline_local/please_inline.rs

```diff
@@ -14,6 +14,5 @@ pub mod a {
 pub mod b {
     // @hasraw - 'pub use foo::'
     // @!has please_inline/b/struct.Foo.html
-    #[feature(inline)]
```

Comment:
Because of this:

```
error[E0518]: attribute should be applied to function or closure
  --> /home/imperio/rust/rust/src/test/rustdoc/inline_local/please_inline.rs:17:5
   |
17 |     #[inline(feature)]
   |     ^^^^^^^^^^^^^^^^^^
18 |     pub use foo::Foo;
   |     ----------------- not a function or closure
```

This error is now emitted because I needed to enable the `UNUSED_ATTRIBUTES` lint in rustdoc. Before it was silently ignored.